aea/cli/logout.py

```diff
@@ -28,8 +28,17 @@
 
 @click.command(name="logout", help="Logout from Registry account.")
 def logout():
-    """Logout from Registry account."""
+    """Logout from Registry account command."""
     click.echo("Logging out...")
+    do_logout()
+    click.echo("Successfully logged out.")
+
+
+def do_logout() -> None:
```

Comment:
private?

Reply:
Just by the analogy with other `do_` methods made it public.